src/Naming/PathNamerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Naming;
+
+/**
+ * @author Paul Le Corre <paul@lecorre.me>
+ */
+interface PathNamerInterface
+{
+    public function namePath(string $resourceShortName) : string;
```

Comment:
Shouldn't we use naming like Symfony's `UrlGeneratorInterface` ? I would prefer to see `generateRootPath` for instance. Or maybe completely extracting the Resource url generation in this class.